client-app/ui-kit/components/atoms/text-area/vc-text-area.vue

```diff
@@ -9,7 +9,7 @@
       class="rounded w-full leading-tight p-3 border focus:border-gray-400 outline-none"
       :class="$attrs.class"
     />
-    <p v-if="counter" class="absolute text-xs text-gray-300 bottom-3.5 right-3">{{ symbolsCount }}</p>
+    <p v-if="counter" class="text-xs text-[color:var(--color-danger)] self-end">{{ symbolsCount }}</p>
```

Comment:
Maybe it's not good to always show counter with danger color.
Please, make this logic instead:
- show counter in gray by default
- show counter in danger color if there is less then 10 symbols left

Reply:
done